service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/auth/SymmetricKey.java

```diff
@@ -34,6 +29,7 @@ public SymmetricKey()
     {
         try
         {
+            String encryptionMethod = "AES";
```

Comment:
I observe a pattern where we have moved constants to local variables. Is that a warning because it has not been used more than once?

Reply:
It's both. Warning because of the usage only in one place.

Reply:
Would making the field be "private static final" also fix this warning? I also see this as opposite to how we typically handle these magic strings

Reply:
Good q: I don't know, will try it out and see. Is it more of a preference thing though?
I also thought about how we follow it currently however I saw this being used only once and made the change accordingly.

Reply:
+1 to making this "private static final", it would be better to have those constant defined at class level

Reply:
This is only used at one place and since the warning. I don't prefer suppressing the warning here unless I am missing any other benefits of having it defined at class level. Otherwise I'll leave this as it is.

Reply:
private static works, thanks @timtay-microsoft